### DISCUSSION TITLE
SOv2.3.61 Intel.dat and Salt mapping Issue

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#5101] [Unsupported Versions] [answered]
After following the instructions for intel.dat modifications I found a couple of issues that prevent intel data from being generated. I've checked both reporter.log and intel.log with no results. It seems that after running a high state to force salt to update local.zeek config in the (zeek docker) the file is unsuccessfully appended. However, salt does indeed pull the intel.dat file into (zeek docker) in the /opt/zeek/ share/zeek/policy/intel/ directory. 

My question is this... How can I successfully append the local.zeek configuration in (zeek docker) to correctly reflect the location of intel.dat? Salt seems to pull intel.dat into docker but will not append local.zeek config. Screenshots below.

By they way... The first screenshot below shows local.zeek config in (zeek docker) maps to intel.dat file to /opt/zeek/share/zeek/site/ which is not correct once salt finishes its highstate.  This is a standalone instance for testing purposes before I move to Distributed setup.

/opt/so/conf/zeek/local.zeek
![image](https://user-images.githubusercontent.com/70167373/128564410-1ca9f17f-afc3-49e6-a1fe-ff03c2338601.png)

(zeek docker) After highstate run
/opt/zeek/share/zeek/site/
![image](https://user-images.githubusercontent.com/70167373/128564432-55bd535d-a877-48e3-b5ad-70a057f596b8.png)

(zeek docker) After highstate run
/opt/zeek/share/zeek/policy/intel/
![image](https://user-images.githubusercontent.com/70167373/128564703-13592b81-b9fb-4bde-be97-585ad1e34708.png)

Update:

I figured it out. you have to place intel.dat in /opt/so/saltstack/local/salt/zeek/policy/intel/
![image](https://user-images.githubusercontent.com/70167373/128567822-6f909a6a-5ac8-424e-8ec6-fe63e8d6ff23.png)

Then you edit /opt/so/saltstack/default/pillar/zeek/init.sls

![image](https://user-images.githubusercontent.com/70167373/128567961-3e4f5f7c-7bfa-433e-b853-3bbaf531c0ee.png)

Then run highstate
![image](https://user-images.githubusercontent.com/70167373/128568036-6514efc0-5b79-4839-8133-47fc15cb133a.png)

**Comment by dougburks** [comment, marked as answer]
You shouldn't have to do all that. I've updated the documentation at https://docs.securityonion.net/en/2.3/zeek.html#intel. Hope that helps!

**Reply by ColeVan** [reply]
Hey Doug! 

Thanks for the response. After fumbling around and understanding the correct directory mappings as well as how salt orchestras everything I finally have a firm grasp on how to properly accomplish adding intel to my intel.dat. The updated docs really help. Thanks for assistance with the matter!

Cole
